modules/RTC/JitsiRemoteTrack.js

```diff
@@ -45,11 +48,12 @@ function JitsiRemoteTrack(
             // Nothing to do if the track is inactive.
         },
         mediaType,
-        videoType,
-        ssrc);
+        videoType);
     this.rtc = rtc;
+    this.ssrc = ssrc;
     this.ownerEndpointId = ownerEndpointId;
     this.muted = muted;
+    this.isP2P = isP2P;
```

Comment:
Hum, so generally I think a track shouldn't be instructed where it is. Is there no way of knowing if the conference is in P2P mode so the track could query it?